scalafix-tests/src/main/scala/scalafix/test/FqnRewrite.scala

```diff
@@ -26,3 +26,12 @@ object LambdaRewrites {
   }
 
 }
+
+case object FqnRewrite3 extends Rewrite {
```

Comment:
Can we also give this rewrite a more descriptive name? For example, `PatchTokenWithEmptyRange`. The `FqnRewrite` names are for testing the classloading feature.